Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs

```diff
@@ -494,13 +538,15 @@ public override void OnWindowAttributesChanged(WindowManagerLayoutParams @params
 
 			_isFullScreen = !_isFullScreen;
 
+			_statusBarUnderlay.Visibility = _isFullScreen ? ViewStates.Gone : ViewStates.Visible;
```

Comment:
Need to toggle underlay visibility so content does not draw over it especially if it has a transparent background.